http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -385,9 +381,52 @@ class ClusterMonitorActor(
       _ <- dbRef.inTransaction { _.clusterQuery.clearKernelFoundBusyDate(cluster.id, now) }
       // Record metrics in NewRelic
       _ <- recordMetrics(cluster.status, ClusterStatus.Stopped).unsafeToFuture()
+//      result <- if (cluster.stopAndUpdate) handleStoppedClusterUpdate(cluster, googleInstances) else Future.successful(ShutdownActor(RemoveFromList(cluster)))
     } yield ShutdownActor(RemoveFromList(cluster))
   }
 
+  private def handleStoppedClusterUpdate(cluster: Cluster, googleInstances: Set[Instance]): Future[ClusterMonitorMessage] = {
```

Comment:
this isn't currently used atm, but will be if product wants to enable the case where numWorkers and machineType are updated at the same time on a running cluster, so leaving it for now